lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -100,15 +96,12 @@ def attributes_for_serializer(serializer, options)
 
         def resource_object_for(serializer, options)
           options[:fields] = @fieldset && @fieldset.fields_for(serializer)
-          options[:required_fields] = [:id, :json_api_type]
```

Comment:
So, we don't need required fields because they're the `{ id: '', type: ''}` in the resource identifier object method?